src/components/shared/FiveStarRatingWidget.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Ratings from 'react-ratings-declarative';
+import PropTypes from 'prop-types';
+
+const FiveStarRatingWidget = props => {
+  const { rating, ...otherProps } = props;
+  return (
+    <Ratings rating={rating} {...otherProps}>
+      <Ratings.Widget />
+      <Ratings.Widget />
+      <Ratings.Widget />
+      <Ratings.Widget />
+      <Ratings.Widget />
+    </Ratings>
+  );
+};
+
+FiveStarRatingWidget.propTypes = {
+  widgetRatedColors: PropTypes.string,
+  widgetDimensions: PropTypes.string,
+  widgetSpacings: PropTypes.string,
+  rating: PropTypes.number,
+};
+
+FiveStarRatingWidget.defaultProps = {
+  widgetRatedColors: '#ffbb28',
+  widgetDimensions: '20px',
+  widgetSpacings: '0px',
+};
```

Comment:
@akshatnitd I have initialised these common props using `defaultProps`, now the rating widget in the feedback card need to pass only `rating` as props except the widget which involves `changeRating` which which uses some other styles such as `widgetHoverColor` that can be passed as well and will be accessed from `otherProps`.